src/test/tree/StrategyManangerUnit.tree

```diff
@@ -0,0 +1,89 @@
+StrategyManagerUnit.t.sol
+├── initialize
+│   └── when initialize called again
+│       └── it should revert
+├── depositIntoStrategy()
+│   ├── when deposit successfully
+│   │   └── shares should increase and stakerStrategyListLength should increase (deposit new strategy)
+│   ├── when deposit successfully twice
+│   │   └── shares should increase and stakerStrategyListLength should increase (deposit new strategy)
```

Comment:
I think this should be combined into two sub-branches of 'when deposit successfully'
they are 'when the caller does not already have shares in the strategy' and 'when the caller already has shares in the strategy'
technically there is also an edge case of 'when the caller does not already have shares in the strategy' -- it is 'when the caller's strategy list is full (max length)'

Reply:
ahh, the latter part is noted further down. I think that works ?